muranotest/config4.conf

```diff
@@ -0,0 +1,56 @@
+[murano]
+# keystone url
+auth_url = http://cloud.lab.fi-ware.org:4731/v3
+
+auth_port = 4731
+
+# Protocol of the admin Identity API endpoint (http or https).
+# Deprecated, use identity_uri. (string value)
+auth_protocol = http
+
+# Complete public Identity API endpoint (string value)
+auth_uri = http://cloud.lab.fiware.org:4731/v3
+
+
+# API version of the admin Identity API endpoint (string value)
+auth_version = v3
+
+password = XXX
+user = admin
+tenant = admin
+region_name = 'Spain2'
+
+# keystone tenant
+# tenant = admin
+
+# keyname - used for debugging murano-agent
+keyname = demo4
+
+# murano url
+# murano_url = http://127.0.0.1:8082
+
+# Flavor for sanity checks
+# standard_flavor = m1.small
+
+# Flavor for advanced checks
+# advanced_flavor = m1.small
+
+# image for linux services
+linux_image =  base_centos_6
+
+conflang_image = base_centos_6 
+
+instance_type=io.murano.resources.ConfLangInstance
+
+
+# image for docker applications
+# docker_image = debian-8-docker.qcow2
+
+# image for kubernetes applications
+# kubernetes_image = ubuntu14.04-x64-kubernetes.qcow2
+
+# image for windows services
+# windows_image = default_windows
+
+# image for hdp sandbox
```

Comment:
what is the difference between config2.conf, config3.conf, config4.conf?

If they are the same why do you need 3 different files?